liquidity/liquidity.go

```diff
@@ -182,7 +279,13 @@ func (m *Manager) SetParameters(params Parameters) error {
 // a reference, we still need to clone the contents of maps.
 func cloneParameters(params Parameters) Parameters {
 	paramCopy := Parameters{
-		FailureBackOff: params.FailureBackOff,
+		FailureBackOff:             params.FailureBackOff,
+		MaximumPrepay:              params.MaximumPrepay,
+		MaximumMinerFee:            params.MaximumMinerFee,
+		MaximumSwapFeePPM:          params.MaximumSwapFeePPM,
+		MaximumRoutingFeePPM:       params.MaximumRoutingFeePPM,
+		MaximumPrepayRoutingFeePPM: params.MaximumPrepayRoutingFeePPM,
+		ConfTarget:                 params.ConfTarget,
 		ChannelRules: make(map[lnwire.ShortChannelID]*ThresholdRule,
 			len(params.ChannelRules)),
 	}
```

Comment:
You could also start off with `paramCopy := params` and then just clone the `ChannelRules`.